admin/src/containers/Users/UsersContext.test.js

```diff
@@ -146,6 +147,27 @@ describe('UsersProvider', () => {
     );
   });
 
+  it('Should toast.error when the API gives a bad response', async () => {
+    global.fetch = jest.fn().mockImplementationOnce(() =>
+      Promise.resolve({
+        status: 404,
+        json: () => ({ error: 'Error message' })
+      })
+    );
+
+    toast.error = jest.fn();
+
+    const { component } = setupShallow();
+
+    try {
+      await component.instance().deleteUser(4);
+    } catch (e) {
+      //not-empty
+    }
+
+    expect(toast.error).toHaveBeenCalledWith('An error occurred: Error message');
```

Comment:
I would also test for `toast.success` to not have been called. I kind of think there's something odd happening there when catching errors on deleting users. I think that you'd have to do `toast.error = jest.fn();` in `beforeEach` in such case.